JSDemos/Demos/DataGrid/BatchUpdateRequest/Vue/App.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <DxDataGrid
+    id="gridContainer"
+    key-expr="OrderID"
+    :data-source="dataSource"
+    :show-borders="true"
+    :remote-operations="true"
+    :repaint-changes-only="true"
+    @saving="onSaving"
+  >
+    <DxEditing
+      mode="batch"
+      :allow-adding="true"
+      :allow-deleting="true"
+      :allow-updating="true"
+    />
+    <DxColumn data-field="OrderID" :allow-editing="false"></DxColumn>
+    <DxColumn data-field="ShipName"></DxColumn>
+    <DxColumn data-field="ShipCountry"></DxColumn>
+    <DxColumn data-field="ShipCity"></DxColumn>
+    <DxColumn data-field="ShipAddress"></DxColumn>
+    <DxColumn data-field="OrderDate" data-type="date"></DxColumn>
+    <DxColumn data-field="Freight"></DxColumn>
+  </DxDataGrid>
+</template>
+<script>
+import { DxDataGrid, DxColumn, DxEditing } from "devextreme-vue/data-grid";
+import { createStore } from "devextreme-aspnet-data-nojquery";
+
+const URL = "https://js.devexpress.com/Demos/Mvc/api/DataGridWebApi";
+
+const dataSource = createStore({
+  key: "OrderID",
+  loadUrl: `${URL}/Orders`,
+  onBeforeSend: (method, ajaxOptions) => {
+    ajaxOptions.xhrFields = { withCredentials: true };
+  }
+});
+
+export default {
+  components: {
+    DxDataGrid,
+    DxColumn,
+    DxEditing
+  },
+  data() {
+    return {
+      dataSource
+    };
+  },
+  created() {
+    this.onSaving = this.onSaving.bind(this);
+  },
```

Comment:
_bind_ is not necessary. Vue does it under the hood when you declare a method in the **methods** section.